framework/auth/cas.py

```diff
@@ -149,6 +149,20 @@ def _parse_profile(self, raw):
             resp.attributes[attribute] = data['attributes'][attribute]
         return resp
 
+    def revoke_application_tokens(self, client_id, client_secret):
```

Comment:
I was thinking more about unit tests, e.g. mocking out the response from the CAS server, then making sure that response is handled as expected.